smarts/core/agent_interface.py

```diff
@@ -102,6 +102,13 @@ class Accelerometer:
     pass
 
 
+@dataclass
+class Goalpoints:
+    """ Observation of nearby goalpoints """
```

Comment:
Can we remove the extra spaces?

Reply:
Certainly, for some reason I cannot stop the habit of padding these doc-strings.